src/m365/tenant/commands/status/status-list.ts

```diff
@@ -54,14 +70,29 @@ class TenantStatusListCommand extends SpoCommand {
       }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
   }
 
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-w, --workload [workload]	',
+        description: 'Allows retrieval of current service status for only one particular service. If not provided, list the current service status of all services will be returned.'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
   public commandHelp(args: {}, log: (help: string) => void): void {
     log(vorpal.find(this.name).helpInformation());
     log(
       `  Examples:
```

Comment:
> We should add a remark how to retrieve the workload name, since it's not the display name visible in the default output

That totally make sense @waldekmastykarz to give how do we get the workload details. This make users to know the Workload filter.
Another query, not sure it is better than what we have. Do you think it make sense to update the help with the command `tenant service list` once the command is in place after the closure of issue #1740 . Or this way would be better?

Reply:
Yes, we could definitely add a reference to it, as another way to get the list of services/workloads. Good idea! 👍